src/app/api/services/dot-roles/dot-roles.service.ts

```diff
@@ -21,16 +21,16 @@ export class DotRolesService {
      * @returns Observable<DotRole[]>
      * @memberof DotRolesService
      */
-    get(roleId: string): Observable<DotRole[]> {
+    get(roleId: string, roleHierarchy: boolean): Observable<DotRole[]> {
```

Comment:
Why don't we leave a default here?
```suggestion
    get(roleId: string, roleHierarchy = true): Observable<DotRole[]> {
```

Reply:
this is taken from the workflow given by the endpoint.